tests/test_contrib_orcid.py

```diff
@@ -128,6 +132,7 @@ def test_authorized_signup(userprofiles_app, example_orcid, orcid_bio):
             id=example_data['orcid']
         ).one()
         # FIXME see contrib/orcid.py line 167
+        assert user.profile.full_name == 'Josiah Carberry'
```

Comment:
Can you check it locally with https://github.com/inveniosoftware/invenio-userprofiles/pull/65 ?